src/components/Quiz/QuizWidget.tsx

```diff
@@ -0,0 +1,410 @@
+        >
+          <Icon
+            as={
+              !showAnswer
+                ? Trophy
+                : currentQuestionAnswerChoice?.isCorrect
+                ? Correct
+                : Incorrect
+            }
+            fontSize="1.75rem"
+            color="neutral"
+          />
+        </Circle>
+        {quizData ? (
+          <>
+            <Center>
+              <Text fontStyle="normal" fontWeight="700" color="primaryHover">
+                {quizData.title}
+              </Text>
+            </Center>
+            {/* Progress bar */}
+            <Center gap={PROGRESS_BAR_GAP} marginBottom={6}>
+              {quizData.questions.map(({ id }, index) => {
+                /* Calculate width percent based on number of questions */
+                const width = `calc(${Math.floor(
+                  100 / quizData.questions.length
+                )}% - ${PROGRESS_BAR_GAP})`
+                return (
+                  <Container
+                    key={id}
+                    bg={progressBarBackground(index)}
+                    h="4px"
+                    w={width}
+                    maxW={`min(${width}, 2rem)`}
+                    marginInline={0}
+                    p={0}
+                  />
+                )
+              })}
+            </Center>
+            <Center>
+              {showResults ? (
+                <QuizSummary
+                  correctCount={correctCount}
+                  questionCount={quizData.questions.length}
+                />
+              ) : (
+                <QuizRadioGroup
+                  questionData={quizData.questions[currentQuestionIndex]}
+                  showAnswer={showAnswer}
+                  handleSelection={handleSelection}
+                  selectedAnswer={selectedAnswer}
+                />
+              )}
+            </Center>
+            <Center mt={8}>
+              <Flex
+                gap={6}
+                flex={{ base: 1, sm: "unset" }}
+                direction={{ base: "column", sm: "row" }}
+                sx={{
+                  button: { width: { base: "100%", sm: "fit-content" } },
+                }}
+              >
+                {showAnswer &&
+                  currentQuestionAnswerChoice &&
+                  !currentQuestionAnswerChoice.isCorrect && (
+                    <Button
+                      onClick={handleRetryQuestion}
+                      variant="outline-color"
+                    >
+                      Try again
+                    </Button>
+                  )}
+                {showResults ? (
+                  <>
+                    <Button
+                      leftIcon={<Icon as={FaTwitter} />}
+                      onClick={handleShare}
+                    >
+                      Share results
+                    </Button>
+                    {score < 100 && (
+                      <Button onClick={initialize}>Try again</Button>
+                    )}
```

Comment:
@minimalsm If we decide we don't want to hide this button with 100%, just commit this code suggestion:
```suggestion
                    <Button onClick={initialize}>Try again</Button>
```